tests/kafkatest/services/kafka/kafka.py

```diff
@@ -1010,84 +1010,68 @@ def kafka_metadata_quorum_cmd(self, node, kafka_security_protocol=None, use_cont
-            bootstrap_server_or_zookeeper = "--zookeeper %s" % (self.zk_connect_setting())
-            skip_optional_security_settings = True
-        else:
-            if kafka_security_protocol is None:
-                # it wasn't specified, so use the inter-broker security protocol if it is PLAINTEXT,
-                # otherwise use the client security protocol
-                if self.interbroker_security_protocol == SecurityConfig.PLAINTEXT:
-                    security_protocol_to_use = SecurityConfig.PLAINTEXT
-                else:
-                    security_protocol_to_use = self.security_protocol
+        if kafka_security_protocol is None:
+            # it wasn't specified, so use the inter-broker security protocol if it is PLAINTEXT,
+            # otherwise use the client security protocol
+            if self.interbroker_security_protocol == SecurityConfig.PLAINTEXT:
+                security_protocol_to_use = SecurityConfig.PLAINTEXT
             else:
-                security_protocol_to_use = kafka_security_protocol
-            bootstrap_server_or_zookeeper = "--bootstrap-server %s" % (self.bootstrap_servers(security_protocol_to_use, offline_nodes=offline_nodes))
-            skip_optional_security_settings = security_protocol_to_use == SecurityConfig.PLAINTEXT
-        if skip_optional_security_settings:
-            optional_jass_krb_system_props_prefix = ""
-            optional_command_config_suffix = ""
+                security_protocol_to_use = self.security_protocol
         else:
-            # we need security configs because aren't going to ZooKeeper and we aren't using PLAINTEXT
-            if (security_protocol_to_use == self.interbroker_security_protocol):
-                # configure JAAS to provide the broker's credentials
-                # since this is an authenticating cluster and we are going to use the inter-broker security protocol
-                jaas_conf_prop = KafkaService.ADMIN_CLIENT_AS_BROKER_JAAS_CONF_PROPERTY
-                use_inter_broker_mechanism_for_client = True
-            else:
-                # configure JAAS to provide the typical client credentials
-                jaas_conf_prop = KafkaService.JAAS_CONF_PROPERTY
-                use_inter_broker_mechanism_for_client = False
-            # We are either using SASL (SASL_SSL or SASL_PLAINTEXT) or we are using SSL
-            using_sasl = security_protocol_to_use != "SSL"
-            optional_jass_krb_system_props_prefix = "KAFKA_OPTS='-D%s -D%s' " % (jaas_conf_prop, KafkaService.KRB5_CONF) if using_sasl else ""
-            optional_command_config_suffix = " --command-config <(echo '%s')" % (self.security_config.client_config(use_inter_broker_mechanism_for_client = use_inter_broker_mechanism_for_client))
+            security_protocol_to_use = kafka_security_protocol
+        bootstrap_server_or_zookeeper = "--bootstrap-server %s" % (self.bootstrap_servers(security_protocol_to_use, offline_nodes=offline_nodes))
```

Comment:
Should we now rename this variable?
Also there are still plenty of ZooKeeper related comments that don't make sense anymore, we should probably delete them. WDYT?

Reply:
> Should we now rename this variable?

+1

> Also there are still plenty of ZooKeeper related comments that don't make sense anymore, we should probably delete them. WDYT?

+1, but we should limit the cleanup to only the comments in kafka.py to avoid creating a large patch.

Reply:
> +1, but we should limit the cleanup to only the comments in kafka.py to avoid creating a large patch.

Agreed